DifferentiationInterface/ext/DifferentiationInterfaceEnzymeExt/utils.jl

```diff
@@ -10,11 +10,11 @@ to_val(::DI.BatchSizeSettings{B}) where {B} = Val(B)
 
 ## Annotations
 
-function get_f_and_df(f::F, ::AutoEnzyme{M,Nothing}, ::Val{B}=Val(1)) where {F,M,B}
+function get_f_and_df(f::F, ::AutoEnzyme{M,Nothing}; (::Val{B})=Val(1)) where {F,M,B}
```

Comment:
this automatic formatting changes the semantics, turning a positional argument into a keyword argument

Reply:
this could be disabled with separate_kwargs_with_semicolon=false

I think this is the behaviour one would want for the option though, no? It seems if it didn't make this transformation previously it would be a bug.

Reply:
or is it such that ::Val{B}=Val(1) is not considered a keyword argument but val::Val{B}=Val(1) is?

Reply:
No, previously it wasn't a bug. It was an unnamed positional argument, for which I'm only interested in the type (think of it as a Holy trait).
Besides, @MilesCranmer seems to say that the same behavior can be observed for named positional arguments, which is also problematic.

Reply:
Yes. Here’s an example in my codebase:

```diff
function check_constraints(
     tree::AbstractExpressionNode,
     options::AbstractOptions,
-    maxsize::Int,
+    maxsize::Int;
     cursize::Union{Int,Nothing}=nothing,
 )::Bool
 ```

This means that doing a function call `check_constraints(tree, options, maxsize, cursize)` will now throw an error when previously it worked.